scripts/test.sh

```diff
@@ -57,12 +57,16 @@ function run_flake8 {
   flake8 scripts
 }
 
-# Default test function, ran by `npm test`.
+# Default test function, run by `npm test`.
 function run_tests {
   find pages* -name '*.md' -exec markdownlint {} +
   tldr-lint ./pages
   for f in ./pages.*; do
-    tldr-lint --ignore "TLDR003,TLDR004,TLDR005,TLDR015,TLDR104" "${f}"
+    if [[ $f == *zh* || $f == *zh_TW* ]]; then
```

Comment:
I think it is better to leave it as a wildcard here so that new languages aren't ignored (we can exclude them if it adds a trailing space like Chinese pages).

Reply:
Yep

Reply:
Don't they have special characters, i.e in languages like Spanish/French the first character might be a special one like ṗ not excluding the first character should be a capital one would be problomatic.

Reply:
> But that's only TLDR003. TLDR004, TLDR015, TLDR104 are also ignored for too many languages.

I think TLDR004 detects more info link line too (we need to remove it first before adding here as we are discussing removing periods behind more information link line).

TLDR015 is same as TLDR003 with only difference being it works on example descriptions.

TLDR104 on the other hand was specifically handcrafted for the English page in https://github.com/tldr-pages/tldr-lint/commit/a9b22cc6905c27d9abba8b363300c0ff6c869de1, we need to rework it as a whole if we need to extend it to other languages I think.

Reply:
I will unresolved this conversation for historical reference, to look up easily in future.